config/sync/core.extension.yml

```diff
@@ -15,6 +15,7 @@ module:
   ctools: 0
   datetime: 0
   dblog: 0
+  devel: 0
```

Comment:
Can we remove this from this PR, I don't want to enable devel. You can just delete this line from the file and commit.